### PR TITLE
Relations Combined

### PR DESCRIPTION
**Pull request by muralithiyagarajan**

### CODE REVIEW

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 1]
```diff
@@ -789,13 +789,238 @@ FROM motorola_cm."cell_x_export" t1
...
+hex_to_int(REPLACE(t5.data->>'LAC','H''','')) as "SRV LAC",
+t3.data->>'CELLID'as "SRV  CELLID",
+t5.data->>'CELLNAME' as "SRV  Cell Name",
+'3G' as "NBR TECH",
+hex_to_int(REPLACE(t4.data->>'LAC','H''','')) as "NBR LAC",
+(t3.data->>'NCELLID')::INTEGER as "NBR CELLID" ,
+t4.data->>'CELLNAME' as "NBR Cellname"
+from huawei_cm."UINTRAFREQNCELL" t3
+inner join huawei_cm."UEXT3GCELL" t4 on t1.data->>'FILENAME'=t2.data->>'FILENAME' and t4.data->>'NCELLID' = t4.data->>'CELLID' and t3.data->>'RNCID' <> t4.data->>'NRNCID'
+inner join huawei_cm."UCELL" t5 on t1.data->>'FILENAME'=t5.data->>'FILENAME' and t3.data->>'CELLID'= t5.data->>'CELLID'
```
Comment (erssebaggala) [inline db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: This query seems to have an error. It is referencing **t1.data** . It does not exist.
```
inner join huawei_cm."UCELL" t5 on **t1**.data->>'FILENAME'=t5.data->>'FILENAME' and t3.data->>'CELLID'= t5.data->>'CELLID'
```

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 2]
```diff
@@ -789,13 +789,238 @@ FROM motorola_cm."cell_x_export" t1
...
+'3G' as "SRV TECH",
+(t3.data->>'refULocationArea')::INTEGER as "SRV LAC",
+t1.data->>'cid' as "SRV  CELLID",
+t3.data->>'userLabel' as "SRV  Cell Name",
+'3G' as "NBR TECH",
+(t2.data->>'refULocationArea')::INTEGER as "NBR LAC",
+(t1.data->>'ncid')::INTEGER as "NBR CELLID" ,
+t2.data->>'userLabel' as "NBR Cellname"
+from zte_cm."UtranRelation" t1
+inner join zte_cm."UtranCellFDD" t2 on t1.data->>'FILENAME'=t2.data->>'FILENAME' and t1.data->>'DataType'=t3.data->>'DataType' and t2.data->>'cid' = t1.data->>'ncid' and t1.data->>'rncid' = t1.data->>'rncid'
```
Comment (erssebaggala) [inline db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: `and t1.data->>'DataType'=t3.data->>'DataType' ` should be `and t1.data->>'DataType'=t2.data->>'DataType'`

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js [hunk 3]
```diff
@@ -789,13 +789,238 @@ FROM motorola_cm."cell_x_export" t1
...
+'3G' as "SRV TECH",
+(t3.data->>'refULocationArea')::INTEGER as "SRV  LAC",
+t1.data->>'cid' as "SRV  CELLID",
+t3.data->>'userLabel' as "SRV  Cell Name",
+'3G' as "NBR TECH",
+(t2.data->>'lac')::INTEGER as "NBR LAC",
+(t1.data->>'ncid')::INTEGER as "NBR CELLID",
+t2.data->>'userLabel' as "NBR Cellname"
+from zte_cm."UtranRelation" t1
+inner join zte_cm."ExternalUtranCellFDD" t2 on t1.data->>'FILENAME'=t2.data->>'FILENAME' and t1.data->>'DataType'=t3.data->>'DataType' and t2.data->>'ncid' = t1.data->>'ncId' and t1.data->>'rncid' <> t1.data->>'rncid'
```
Comment (erssebaggala) [inline db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: `and t1.data->>'DataType'=t3.data->>'DataType'` should be `and t1.data->>'DataType'=t2.data->>'DataType'`

You are reference t3 before it is joined!

Comment (muralithiyagarajan) [reply db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: Corrected

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js (hunk 2 above)
Comment (muralithiyagarajan) [reply db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: Corrected

File: db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js (hunk 1 above)
Comment (muralithiyagarajan) [reply db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js]: corrected

**Pull request merged**